src/Microsoft.Data.SqlClient/add-ons/AzureKeyVaultProvider/Microsoft.Data.SqlClient.AlwaysEncrypted.AzureKeyVaultProvider.csproj

```diff
@@ -15,6 +15,8 @@
     <BuildProjectReferences>false</BuildProjectReferences>
     <PackageLicenseExpression>MIT</PackageLicenseExpression>
     <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
+    <IsTrimmable Condition="'$(TargetGroup)'=='netcoreapp'">true</IsTrimmable>
```

Comment:
Technically this is implied by setting `IsAotCompatible`. I think having both properties is slightly clearer until we have end-to-end support for trimming, but that's purely a matter of taste.